charts/agent-stack-k8s/values.yaml

```
@@ -10,3 +10,6 @@ resources:
   requests:
     cpu: 100m
     memory: 100Mi
+
+labels:
+  user: "user"
```

Given that the default in the JSON schema is `{}`, I think we should do that here as well.
```suggestion
labels: {}
```
Also, this does not fit the JSON schema you've specified. Attempting to deploy it will give:
```
Error: UPGRADE FAILED: values don't meet the specifications of the schema(s) in the following chart(s):
agent-stack-k8s:
- labels.user: Invalid type. Expected: object, given: string
```

But as per my comment above, I don't think the JSON schema is valid for Kubernetes.